deps/wazuh_testing/wazuh_testing/remote.py

```diff
@@ -35,14 +34,14 @@
 data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
 
 
-def new_agent_group(group_name=DEFAULT_TESTING_GROUP_NAME, configuration_file='agent.conf'):
+def new_agent_group(group_name=DEFAULT_TESTING_GROUP_NAME, configuration_file=REMOTE_AGENT_CONF):
```

Comment:
```suggestion
def new_agent_group(group_name=DEFAULT_TESTING_GROUP_NAME, configuration_file=shared.conf):
```
Or create a variable in the `__init__.py` to store the name of the remote configuration file